boards/xenvm-qemu.overlay

```diff
@@ -0,0 +1,57 @@
+
+#include <mem.h>
+
+/ {
+  	fstab {
+		compatible = "zephyr,fstab";
+		storage: storage {
+			compatible = "zephyr,fstab,littlefs";
+			mount-point = "/lfs";
+			partition = <&storage_partition>;
+			automount;
+			read-size = <16>;
+			prog-size = <16>;
+			cache-size = <64>;
+			lookahead-size = <32>;
+			block-cycles = <512>;
+		};
+	};
+	flashcontroller0: flashcontroller {
+		compatible = "zephyr,sim-flash";
+		label = "FLASH_SIMULATOR";
```

Comment:
some notes from Zephyr documentation:
```
Earlier versions of Zephyr made frequent use of the label property, which is distinct from the 
tandard node label. Use of the label property in new devicetree bindings, as well as use of the
DT_LABEL macro in new code, are actively discouraged. Label properties continue to persist
for historical reasons in some existing bindings and overlays, but should not be used in new
bindings or device implementations.
```

moreover, I have seen warnings form `west` that `label` property is a deprecated during compilation of a project where I had DTS labels.